python/lsst/ts/audiotrigger/read_serial_temp_scanner.py

```diff
@@ -37,7 +38,7 @@
 
 
 async def callback(data):
-    SerialTemperatureScanner.data = data
+    setattr(SerialTemperatureScanner, "data", data)
```

Comment:
Please create a setter method in the `SerialTemperatureScanner` class and pass on the data that way.